apps/client/src/intros/ZonnepanelenIntro.js

```diff
@@ -1,41 +1,14 @@
-import { Heading, Paragraph } from "@datapunt/asc-ui";
+import { Paragraph } from "@datapunt/asc-ui";
 import React from "react";
 
-import { ListItem, OrderedList } from "../atoms";
-import ListInsideOrderedList from "../components/ListInsideOrderedList";
+import LocationIntro from "./shared/LocationIntro";
 
 export default () => (
   <>
     <Paragraph gutterBottom={20}>
       U kunt deze vergunningcheck gebruiken voor zonnepanelen en zonneboiler.
     </Paragraph>
-    <Heading forwardedAs="h3">Hoe het werkt:</Heading>
-    <OrderedList>
-      <ListItem>
-        U voert op de volgende pagina eerst het adres van het gebouw in.
-      </ListItem>
-      <ListItem>
-        Vervolgens krijgt u te zien:
-        <ListInsideOrderedList variant="bullet">
-          <ListItem>of het gebouw een monument is.</ListItem>
-          <ListItem>
-            of het gebouw in een beschermd stads- of dorpsgezicht ligt.
-          </ListItem>
-          <ListItem>welk bestemmingsplan er geldt.</ListItem>
-        </ListInsideOrderedList>
-      </ListItem>
-      <ListItem>
-        Wij stellen u een aantal vragen over het gebouw en de zonnepanelen of de
```

Comment:
Now you're missing this text:
_Wij stellen u een aantal vragen over het gebouw **en de zonnepanelen of de zonneboiler.**_

Reply:
That's correct.